### ISSUE TITLE
Problem z nadaniem paczki na Węgry

### ISSUE THREAD

**Issue by krystianpicaro** [allegro/allegro-api#10146]
Witam mamy problem z nadaniem przesyłki na allegro one. 
większość przechodzi normalnie a czasem mamy błąd z którym nie wiemy jak sobie poradzić . Przesyłam poniżej Jsona i błąd jaki otrzymujemy. 
 Błąd: getParcelStatus insurance.amountNiepoprawna kwota dla zadanej waluty; 
{
  "commandId": "64db202b-4766-4b10-a420-5ed1547b7791",
  "input": {
    "id": null,
    "deliveryMethodId": "0258c744-1a2d-4da7-a8d9-d69fc7ae2d08",
    "credentialsId": null,
    "referenceNumber": "XXX",
    "description": "XXX",
    "sender": {
      "email": "XXX",
      "name": "Biuro Obsługi klienta",
      "company": "XXX",
      "phone": "XXX",
      "street": "XXX",
      "streetNumber": "2E",
      "city": "XXX",
      "state": null,
      "postalCode": "XXX",
      "countryCode": "PL",
      "point": null
    },
    "receiver": {
      "email": "tdsa1hlk93+44e8c4192@allegromail.pl",
      "name": "Péter XXX",
      "company": "koplányi XXX",
      "phone": "+XXX",
      "street": "Űrhajós XXX",
      "streetNumber": "XXX",
      "city": "Vác",
      "state": null,
      "postalCode": "XXX",
      "countryCode": "HU",
      "point": "32551"
    },
    "pickup": null,
    "packages": [
      {
        "type": "PACKAGE",
        "waybill": null,
        "length": {
          "value": "30",
          "unit": "CENTIMETER"
        },
        "width": {
          "value": "20",
          "unit": "CENTIMETER"
        },
        "height": {
          "value": "20",
          "unit": "CENTIMETER"
        },
        "weight": {
          "value": "1",
          "unit": "KILOGRAMS"
        }
      }
    ],
    "insurance": {
      "amount": "9571.00",
      "currency": "HUF"
    },
    "cashOnDelivery": null,
    "labelFormat": "PDF",
    "additionalServices": []
  }
}

Proszę o informację co jest źle przekazane ,że dostajemy taki błąd.

**Comment by PrzemyslawLukanowski** [comment]
Czy to na pewno prawidłowe commandID? Widzę w naszych logach, że dla tego przypadku nie zwróciliśmy błędu, przesyłka utworzyła się prawidłowo.

**Comment by krystianpicaro** [comment]
commandId jest poprawne. W momencie tworzenia paczki otrzymujemy błąd insurance.amount Niepoprawna kwota dla zadanej waluty; 
Ponowiliśmy przed chwilą próbę i efekt ten sam dla commandId  2f9de349-9b0c-405c-8d37-47e3732a6229

**Comment by PrzemyslawLukanowski** [comment]
Tak, dla tego przykładu widzę już komunikat błędu. Zweryfikuję to z zespołem odpowiedzialnym za Wysyłam z Allegro i wrócę z informacją.

**Comment by PrzemyslawLukanowski** [comment]
Spróbuj proszę zmienić kwotę ubezpieczenia, taka by była podzielna przez 5. Z naszej strony zmienimy treść komunikatu, aby jednoznacznie wskazywała przyczynę błędu.

**Comment by krystianpicaro** [comment]
Na to nie wpadliśmy . 
Wszystko poszło , przynajmniej jest odpowiedz dlaczego w niektórych przypadkach działało poprawnie.
